### PR TITLE
Support Request Processor Extensions if Feature Flag turned on.

### PR DESCRIPTION
**Pull request by symorton**
**Problem:**
- Support Request Extensions if feature flag is turned on.
- Add logic to translate JWSToken claims to primitive dictionary of String to Any to support extensions.

**Solution:**
- Add logic to translate JWSToken claims to primitive dictionary of String to Any to support extensions.
- If feature flag is turned on, after initial processing in OpenIdRequestProcessor, use processor extensions.

**Validation:**
- Unit tests pass.

**Type of change:**
- [X] Feature work
- [ ] Bug fix
- [ ] Documentation
- [ ] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

**Risk**:
- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [X] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [ ] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

**Work Item links**:
Please include here links for this work item, or deferred work, or related work. E.g. if the refactoring is too big to fit in this PR, or the localized strings need to be updated later, please link the TODO work items here.

**Documentation Links**:
Please include here links to any related background documentation for this PR.

### CODE REVIEW

File: WalletLibrary/WalletLibrary/Requests/OpenId/OpenIdPresentationRequest.swift [hunk 1]
```diff
@@ -40,6 +40,19 @@ class OpenIdPresentationRequest: VerifiedIdPresentationRequest {
         self.configuration = configuration
     }
     
+    init(partialRequest: VerifiedIdPartialRequest,
```
Comment (logangirvin) [inline WalletLibrary/WalletLibrary/Requests/OpenId/OpenIdPresentationRequest.swift:43]: A pattern Bas found in Android is actually having VerifiedIDRequest inherit from VerifiedIdPartialRequest, so we don't technically need a new data structure for extension's uses.

File: WalletLibrary/WalletLibraryTests/Requests/Processors/OpenIdRequestProcessorTests.swift [hunk 2]
```diff
@@ -83,6 +83,108 @@ class OpenIdRequestProcessorTests: XCTestCase {
...
+        
+        // Arrange
+        let expectedStyle = MockRequesterStyle(requester: "mock requester")
+        let expectedRequirement = MockRequirement(id: "mockRequirement324")
+        let expectedRootOfTrust = RootOfTrust(verified: true, source: "mock source")
+        let expectedContent = PresentationRequestContent(style: expectedStyle,
+                                                         requirement: expectedRequirement,
+                                                         rootOfTrust: expectedRootOfTrust,
+                                                         requestState: "mock state",
+                                                         callbackUrl: URL(string: "https://test.com")!)
```
Comment (logangirvin) [inline WalletLibrary/WalletLibraryTests/Requests/Processors/OpenIdRequestProcessorTests.swift]: Can we use a domain we control or a non-existent domain?

**Review by logangirvin — APPROVED** [review]

File: WalletLibrary/WalletLibrary/Requests/OpenId/OpenIdPresentationRequest.swift (hunk 1 above)
Comment (symorton) [reply WalletLibrary/WalletLibrary/Requests/OpenId/OpenIdPresentationRequest.swift:43]: decided off line not to change.

File: WalletLibrary/WalletLibraryTests/Requests/Processors/OpenIdRequestProcessorTests.swift (hunk 2 above)
Comment (symorton) [reply WalletLibrary/WalletLibraryTests/Requests/Processors/OpenIdRequestProcessorTests.swift]: yes changed.

**Pull request merged**